config.py

```diff
@@ -11,6 +11,9 @@ class Config:
     )
     AUTH_REQUIRED = True
     ALLOW_EXPLORER = True
+
+    ELASTICSEARCH_HOST = os.getenv('DM_ELASTICSEARCH_URL', 'localhost:9200')
```

Comment:
The default is to avoid writing a wrapper script for `python application.py update_index`, since it uses the ELASTICSEARCH_HOST. I think we can keep the variable in the scripts, since it's useful as a list of important values that can be set.